pkg/snapstore/abs_snapstore.go

```diff
@@ -172,17 +170,23 @@ func readABSCredentialFiles(dirname string) (*absCredentials, error) {
 
 	for _, file := range files {
 		if file.Name() == "storageAccount" {
```

Comment:
The issue with this approach is that the code now hard codes the name of the file.  This is also quite inconsistent with the following:
```go
filename, isSet := os.LookupEnv(prefixString + absCredentialJSONFile)
```
where we look for an environment variable to get to the file name. We should remove all these inconsistencies later if not as part of this PR.

Reply:
Sure, in a different PR. As I mentioned, etcdbr snapstore package needs some re-looking.